src/manager.rs

```diff
@@ -0,0 +1,194 @@
+use compile::ExecutableDevEnv;
+use parse::Parser;
+use run::{ Execution, ExecutionEvent, Error as RunError };
+use rusqlite;
```

Comment:
Nit: I tend to add empty lines to separate internal modules from std modules from external crates. Also, I put non-Foxlink crates after `std`, so let's match that style.